src/test/java/edu/tamu/app/controller/AdminControllerTest.java

```diff
@@ -11,8 +11,8 @@
 public class AdminControllerTest extends AbstractControllerTest {
 
     @Test
-    public void testSyncDocuments() throws IOException {
-        response = adminController.syncDocuments();
+    public void testSyncDocuments(Long projectId) throws IOException {
+        response = projectController.syncDocuments(projectId);
```

Comment:
The test passes after removing projectId from the method signature and getting the mock project like this instead:

`projectController.syncDocuments(projectRepo.findOne(1L).getId());`